tests/plugins/test_download_remote_source.py

```diff
@@ -36,12 +36,16 @@
 def mock_reactor_config(workflow, insecure=False):
     data = dedent("""\
         version: 1
+        koji:
+            hub_url: /
+            root_url: ''
+            auth: {}
         cachito:
            api_url: 'example.com'
            insecure: {}
            auth:
                ssl_certs_dir: /some/dir
-        """.format(insecure))
+        """.format("{}", insecure))
```

Comment:
When you need a literal `{}` in a formatted string, you can use this:
```python
"a: {{}}, b: {}".format(1)  # -> "a: {}, b: 1"
```
IMO it is more readable.

Note that this appears in other places as well.

Reply:
Wow, thanks :+1: :+1: :+1:

Reply:
Fixed